client/changes_test.go

```diff
@@ -48,12 +48,12 @@ func (cs *clientSuite) TestClientChange(c *check.C) {
 			Summary:   "...",
 			Status:    "Do",
 			Progress:  client.TaskProgress{Done: 0, Total: 1},
-			SpawnTime: time.Date(2016, 04, 21, 1, 2, 3, 0, time.UTC),
-			ReadyTime: time.Date(2016, 04, 21, 1, 2, 4, 0, time.UTC),
+			SpawnTime: time.Date(2016, 0o4, 21, 1, 2, 3, 0, time.UTC),
+			ReadyTime: time.Date(2016, 0o4, 21, 1, 2, 4, 0, time.UTC),
 		}},
 
-		SpawnTime: time.Date(2016, 04, 21, 1, 2, 3, 0, time.UTC),
-		ReadyTime: time.Date(2016, 04, 21, 1, 2, 4, 0, time.UTC),
+		SpawnTime: time.Date(2016, 0o4, 21, 1, 2, 3, 0, time.UTC),
+		ReadyTime: time.Date(2016, 0o4, 21, 1, 2, 4, 0, time.UTC),
```

Comment:
I'd rather see `4` than `0o4` here.

At the same time, the hidden octal `04` was kind nasty, any change is better than no change.